blosc2/core.py

```diff
@@ -1230,7 +1230,11 @@ def compute_chunks_blocks(shape, chunks=None, blocks=None, dtype=np.uint8, **kwa
                     cparams2["filters"][i] = blosc2.Filter.NOFILTER
         else:
             cparams2 = cparams
+        # Do not apply specific tuner
+        aux_tuner = cparams2["tuner"] if "tuner" in cparams2 else blosc2.Tuner.STUNE
+        cparams2["tuner"] = blosc2.Tuner.STUNE
         src = blosc2.compress2(np.zeros(nitems, dtype=f"V{itemsize}"), **cparams2)
+        cparams2["tuner"] = aux_tuner
```

Comment:
I think moving this line one position below would increase readability

Reply:
Fixed in 5d6795a2f591109e1b70a1bca2cbeb6c11844cb5